apps/ws/src/bootstrap.ts

```diff
@@ -1,11 +1,10 @@
-import './config';
 import { NestFactory } from '@nestjs/core';
 import * as Sentry from '@sentry/node';
-import { RedisIoAdapter } from './shared/framework/redis.adapter';
 import { version } from '../package.json';
-
+import { RedisIoAdapter } from './shared/framework/redis.adapter';
 import { AppModule } from './app.module';
 import { CONTEXT_PATH } from './config';
+import './config';
```

Comment:
The config here should be at the top most section of the file because it loads the env variables, so I think we need to exclude this from the eslint check and ignore it in this line